x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/compare_fields_table/helpers.test.tsx

```diff
@@ -230,35 +204,6 @@ describe('helpers', () => {
           expect(screen.queryByTestId('emptyPlaceholder')).not.toBeInTheDocument();
         });
       });
-
-      describe('when `description` is undefined', () => {
-        const withUndefinedDescription = {
-          ...eventCategory,
-          description: undefined, // <--
-        };
-
-        beforeEach(() => {
-          const columns = getEcsCompliantTableColumns();
-          const descriptionRender = columns[3].render;
-
-          render(
-            <TestProviders>
-              <>
-                {descriptionRender != null &&
-                  descriptionRender(withUndefinedDescription.description, withUndefinedDescription)}
-              </>
-            </TestProviders>
-          );
-        });
-
-        test('it does NOT render the `description`', () => {
-          expect(screen.queryByTestId('description')).not.toBeInTheDocument();
-        });
-
-        test('it renders the placeholder', () => {
-          expect(screen.getByTestId('emptyPlaceholder')).toBeInTheDocument();
-        });
-      });
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631423077